addons/message_try_firefox_extension/getExtension.js

```diff
@@ -0,0 +1,3 @@
+((api) => {
+  api.urlOpener.openUrlLabel('downloadExtension');
```

Comment:
Same comment as for `downloadFirefox.js` this could done directly as `api.urlOpener.openUrl('https://foo.bar/baz/...')`